### PR TITLE
Bepro 1499 support multi network on my bounties and multi network design

### PR DESCRIPTION
**Pull request by vhcsilva**

### CODE REVIEW

**Review by moshmage — CHANGES_REQUESTED** [review]

File: components/profile/profile-router.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import PullRequestsPage from "components/profile/pages/pull-requests";
+import VotingPowerPage from "components/profile/pages/voting-power";
+import WalletPage from "components/profile/pages/wallet";
+
+export default function ProfileRouter() {
+  const { asPath, push } = useRouter();
+
+  const Route = (path, page) => ({ path, page });
+  
+  const routes = [
```
[the remark this answers is not in the document]
Comment (vhcsilva) [reply components/profile/profile-router.tsx:19]: What you think of this way ?
```
const routes = [
    Route("/profile", ProfilePage),
    ...
  ];
//...
return <currentRoute.page />;
```

[the remark this answers is not in the document]
Comment (moshmage) [reply components/profile/profile-router.tsx:19]: That works too :)

File: pages/profile/[[...profilePage]].tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { GetServerSideProps } from "next";
```
Comment (moshmage) [inline pages/profile/[[...profilePage]].tsx:1]: You can simply import the same `profile` component you created for `pages/[network]/[chain]/profile/[[...profilePage]].tsx` (or import this one on the previously mentioned)

Comment (moshmage) [inline pages/profile/[[...profilePage]].tsx:1]: You can simply import the same `profile` component you created for `pages/[network]/[chain]/profile/[[...profilePage]].tsx` (or import this one on the previously mentioned)

**Review by moshmage — CHANGES_REQUESTED** [review]
looking good, just a minor detail about the profile pages and we're a-ok

**Review by moshmage — APPROVED** [review]

**Pull request merged**